tests/operators/test_agnostic_load_file.py

```diff
@@ -50,9 +51,11 @@
 from google.cloud import bigquery, storage
 from pandas.util.testing import assert_frame_equal
 
+from astro.constants import DEFAULT_SCHEMA
```

Comment:
@tatiana do you agree with us having this DEFAULT_SCHEMA variable?

Reply:
@dimberman I commented here:
https://github.com/astro-projects/astro/pull/184/files#r825188964